packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -0,0 +1,280 @@
+import * as React from 'react';
+
+import {
+  Box,
+  Button,
+  FieldLabel,
+  Flex,
+  ModalBody,
+  ModalHeader,
+  ModalLayout,
+  SingleSelect,
+  SingleSelectOption,
+  Typography,
+  ModalFooter,
+} from '@strapi/design-system';
+import {
+  CheckPermissions,
+  useAPIErrorHandler,
+  useCMEditViewDataManager,
+  useNotification,
+} from '@strapi/helper-plugin';
+import { Plus } from '@strapi/icons';
+import { isAxiosError } from 'axios';
+import { Formik, Form } from 'formik';
+import { useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import * as yup from 'yup';
+
+import { CreateReleaseAction } from '../../../shared/contracts/release-actions';
+import { PERMISSIONS } from '../constants';
+import { useCreateReleaseActionMutation, useGetReleasesQuery } from '../services/release';
+
+import { ReleaseActionOptions } from './ReleaseActionOptions';
+
+const RELEASE_ACTION_FORM_SCHEMA = yup.object().shape({
+  type: yup.string().oneOf(['publish', 'unpublish']).required(),
+  releaseId: yup.string().required(),
+});
```

Comment:
Do you think makes sense moves this to the shared/validation-schema file? So we keep all the related schemas in the same place

Reply:
For me it doesn't make sense because we will never share this with the server side